.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Proposed changes
+
+Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue using one of the [supported keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in this PR's description or commit message.
+
+### Checklist
+
+Before creating a PR, run through this checklist and mark each as complete:
+
+- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md).
+- [ ] I have signed the [F5 Contributor License Agreement (CLA)](https://github.com/f5/f5-cla/blob/main/docs/f5_cla.md).
+- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works.
+- [ ] If applicable, I have checked that any relevant tests pass after adding my changes.
+- [ ] I have updated any relevant documentation ([`README.md`](/README.md) and/or [`CHANGELOG.md`](/CHANGELOG.md)).
```

Comment:
Do we think we want to maintain a changelog for this project? If we don't, we can remove the Changelog requirement here.